### ISSUE TITLE
Weight class labels in loss criterion

### ISSUE THREAD

**Issue by cegme** [ufdatastudio/mixteclabeling#23]
Trying adding weights in the cross entropy loss to weight errors the errors in the minority class higher. This may help with the imbalance.
https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html#torch.nn.CrossEntropyLoss
